config/config-autoscaler.yaml

```diff
@@ -54,16 +54,28 @@ data:
     # to achieve efficient resource usage (VPA CPU minimum is 300m).
     container-concurrency-target-default: "100"
 
-    # When operating in a stable mode, the autoscaler operates on the
-    # average concurrency over the stable window.
-    stable-window: "60s"
+    # When observed average concurrency during the panic window reaches
+    # target-panic-percentage the target concurrency, the autoscaler
+    # enters panic mode. When operating in panic mode, the autoscaler
+    # scales on the average concurrency over the panic window which is
+    # window-panic-percentage of the stable-window.
+    window-panic-percentage: "10.0"
```

Comment:
Why make this a percentage? Couldn't we just as easily make this a duration setting?

Reply:
I want this to be a percentage because the basic behavior should change when the user specifies just `window`.  E.g. I want the panic window to scale with the stable window.  `window` and `target` are the basic annotations.  Adjusting the panic window's relationship to the stable window is more advanced.

Reply:
Good point, thanks for clarifying that!

Reply:
I also added this rationale in the `register.go` documentation.